cmd/containerboot/main.go

```diff
@@ -731,16 +731,16 @@ func tailscaledConfigFilePath() string {
 		}
 		cv, err := kubeutils.CapVerFromFileName(e.Name())
 		if err != nil {
-			log.Printf("skipping file %q in tailscaled config directory %q: %v", e.Name(), dir, err)
 			continue
 		}
 		if cv > maxCompatVer && cv <= tailcfg.CurrentCapabilityVersion {
 			maxCompatVer = cv
 		}
 	}
 	if maxCompatVer == -1 {
-		log.Fatalf("no tailscaled config file found in %q for current capability version %q", dir, tailcfg.CurrentCapabilityVersion)
+		log.Fatalf("no tailscaled config file found in %q for current capability version %d", dir, tailcfg.CurrentCapabilityVersion)
 	}
-	log.Printf("Using tailscaled config file %q for capability version %q", maxCompatVer, tailcfg.CurrentCapabilityVersion)
-	return path.Join(dir, kubeutils.TailscaledConfigFileName(maxCompatVer))
+	filePath := path.Join(dir, kubeutils.TailscaledConfigFileName(maxCompatVer))
```

Comment:
nit: Not that it needs to work on other operating systems, but looks like we're using the `path` package rather than `filepath`.

```suggestion
	filePath := filepath.Join(dir, kubeutils.TailscaledConfigFileName(maxCompatVer))
```